tests/lib.rs

```diff
@@ -7,15 +7,16 @@ use nom::{
 
 use TokenKind::*;
 
-macro_rules! rule {
-    ($($tt:tt)*) => { nom_rule::rule!(match_text, match_token, $($tt)*) }
+/// Differ these rule macro.
+macro_rules! rule_ {
```

Comment:
Do you think using the same name caused any confusion? Personally, I think the custom rule should rather take the simple name `rule` because it's what we're actually using in the parser frequently.

Reply:
> Do you think using the same name caused any confusion? Personally, I think the custom rule should rather take the simple name `rule` because it's what we're actually frequently using in the parser.

That's ok. I will revoke this PR.